uefi/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "uefi"
+version = "0.17.0"
+authors = ["Gabriel Majeri <gabriel.majeri6@gmail.com>"]
+readme = "README.md"
+edition = "2021"
+exclude = [
+    ".cargo/**",
+    "template/**",
+    "uefi-macros/**",
+    "uefi-services/**",
+    "uefi-test-runner/**",
+    "xtask/**",
+]
```

Comment:
I think `exclude` can be dropped now that `uefi` is in its own directory.

Reply:
good catch!